### PR TITLE
Added CRUD for posts,filters

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
Files to check:-
src/utils/getFilteredData.jsx
src/components/sidebar/leftSidebar.jsx
src/components/sidebar/rightSidebar.jsx
src/features/explore/explore.jsx
src/features/home/home.jsx
src/features/post/post.jsx
src/features/post/postModal.jsx
src/features/post/postTextbox.jsx
src/features/post/postSlice.jsx
src/features/profile/userSlice.jsx

Thanks for reviewing the PR.

### CODE REVIEW

File: src/features/explore/explore.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Box, Flex, Heading, useColorModeValue } from "@chakra-ui/react"
+import { LeftSidebar, RightSidebar } from "components"
+import { Post } from "features/post/post"
+import { useSelector } from "react-redux"
+
+export const Explore = () => {
+    const {allPosts} = useSelector((store)=>store.posts);
+    const explorePosts = [...allPosts].reverse()
+    console.log([...allPosts].reverse())
```
Comment (nayyyhaa) [inline src/features/explore/explore.jsx]: can remove console.log from here

File: src/features/post/postModal.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        }else if(postContent){
+            dispatch(createPost({content:postContent}))
+            setPostContent("")
+            onClose();
+        }else{
+            toast.warning("Post can't be empty!");
+        }
+    }
+    return (
+      <>
```
Comment (nayyyhaa) [inline src/features/post/postModal.jsx]: can remove extra fragment tag

**Review by nayyyhaa — APPROVED** [review]
Nice work! redux slices are well written. Not much to suggest, you can merge the changes. 👍 keep it up!

**Review by vrx29 — APPROVED** [review]
Everything looks good. Nice that you have implemented redux as well in your code. You can merge it

**Pull request merged**